arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3315,50 +3301,53 @@ arangodb::Result hotBackupDBServers(std::string const& backupId, std::string con
     builder.add("allowInconsistent", VPackValue(force));
     builder.add("nrDBServers", VPackValue(dbServers.size()));
   }
-  auto body = std::make_shared<std::string>(builder.toJson());
 
   std::string const url = apiStr + "create";
-  std::vector<ClusterCommRequest> requests;
 
+  network::RequestOptions reqOpts;
+  reqOpts.retryNotFound = false;
```

Comment:
same here. `retryNotFound` is false by default, so it doesn't need to be set. If set here for verboseness, the calls above should also set it.